python/pyspark/ml/tuning.py

```diff
@@ -288,6 +289,195 @@ def copy(self, extra=None):
         return CrossValidatorModel(self.bestModel.copy(extra))
 
 
+class TrainValidationSplit(Estimator, HasSeed):
+    """
+    Train-Validation-Split.
```

Comment:
There needs to be a space here or the docs won't generate correctly. You can check this on your local machine by running `jekyll build` in the docs folder.